### ISSUE TITLE
Record the relative time intervals between each key press?

### ISSUE THREAD

**Issue by whoyawn** [RMPR/atbswp#43]
## Checklist

- [x] I use the latest release of atbswp
- [x] The issue is not existing yet

## Verbose log

```
PASTE VERBOSE LOG HERE
```

### Operating System
Windows 10

### Desktop Environment/Window Manager
Write here

### Python version
3.9

## Description

Currently, each keypress has a consistent delay in between and I'm trying to find a way to record the exact time in between them.

For example, if the script generated could look like this:

```
pyautogui.moveTo(2586, 674)
pyautogui.mouseDown(2531, 466, 'left')
pyautogui.mouseUp(2531, 466, 'left')
time.sleep(1)
pyautogui.keyDown('enter')
pyautogui.keyUp('enter')
pyautogui.keyDown('enter')
timer.sleep(.234) # DELAY
pyautogui.keyUp('enter')
timer.sleep(.333) # DELAY
pyautogui.keyDown('e')
pyautogui.keyDown('n')
```

It seems like the easiest way to do this would be to set a timer in between each `record` call, but if there's an easier way to do this, let me know.

**Comment by borderline23** [comment]
also muose movement speed will be great! thanks for your work

**Comment by bmn001** [comment]
It looks like maybe this has been added?

I see things like "time.sleep(1)" and (2) in my scripts.  Full-second delays are maybe not as granular as OP was hoping for, but it looks like there is support for it now.

**Comment by RMPR** [comment]
Yes, you're right. It was solved, and even made more accurate with #85 but since that commit has been merged, no release was made. I'll close the issue after making the release.
